cmd/create/oidcprovider/cmd.go

```diff
@@ -50,10 +50,21 @@ var Cmd = &cobra.Command{
 	Run: run,
 }
 
+var args struct {
+	oidcEndpointUrl string
+}
+
 func init() {
 	flags := Cmd.Flags()
 
-	ocm.AddClusterFlag(Cmd)
+	flags.StringVar(
```

Comment:
This should be hidden too since users should only be passing this if they're aware of the other hidden flags during cluster creation